### PR TITLE
Fix current Rubocop Issues

### PR DESCRIPTION
**Pull request by mortik**
Should also fix the following Stories:

[SC-10366](https://app.shortcut.com/shipcloud/story/10366/billwerk-gem-fehlende-frozen-string-literal-kommentare-hinzufügen)

### CODE REVIEW

**Comment by ChristianBreitkreutz** [comment]
**L**ooks**G**ood**T**o**M**e

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+        environment:
+          LANG: en_US.UTF-8
+          BUNDLE_DISABLE_VERSION_CHECK: 1
+    steps:
+      - checkout
+      - run:
+          name: Install ruby dependencies
+          command: gem update --system; bin/setup
+      - run:
+          name: Execute specs
```
Comment (SteffenHam) [inline .circleci/config.yml:15]: How about a more descriptive name? Something like `Run Rubocop` maybe?

**Review by SteffenHam — CHANGES_REQUESTED** [review]
I like having Rubocop as an extra step in the CI 👍

File: .circleci/config.yml (hunk 1 above)
Comment (mortik) [inline .circleci/config.yml:15]:
```suggestion
          name: Run Rubocop
```

**Review by SteffenHam — APPROVED** [review]
RTM 👍

**Pull request merged**